pwiz_tools/BiblioSpec/src/BlibUtils.cpp

```diff
@@ -133,6 +133,24 @@ const char* ionMobilityTypeToString(IONMOBILITY_TYPE ionMobilityType)
     }
 }
 
+IONMOBILITY_TYPE parseIonMobilityType(const char* ionMobilityType)
+{
+    if (bal::iequals(ionMobilityType, "none")) return IONMOBILITY_NONE;
+
+    if (bal::iequals(ionMobilityType, "driftTime(msec)")) return IONMOBILITY_DRIFTTIME_MSEC;
+    if (bal::iequals(ionMobilityType, "ms")) return IONMOBILITY_DRIFTTIME_MSEC;
+
+    if (bal::iequals(ionMobilityType, "inverseK0(Vsec/cm^2)")) return IONMOBILITY_INVERSEREDUCED_VSECPERCM2;
+    if (bal::iequals(ionMobilityType, "inverseK0")) return IONMOBILITY_INVERSEREDUCED_VSECPERCM2;
+    if (bal::iequals(ionMobilityType, "1/K0")) return IONMOBILITY_INVERSEREDUCED_VSECPERCM2;
+    if (bal::iequals(ionMobilityType, "Vsec/cm^2")) return IONMOBILITY_INVERSEREDUCED_VSECPERCM2;
+
+    if (bal::iequals(ionMobilityType, "compensation(V)")) return IONMOBILITY_COMPENSATION_V;
+    if (bal::iequals(ionMobilityType, "V")) return IONMOBILITY_COMPENSATION_V;
```

Comment:
Good flexibility - ideally this would contain at least what we support in Skyline's transition list reader, as in pwiz_tools\Skyline\Model\SmallMoleculeTransitionListReader.cs line 982.